.github/workflows/release.yml

```diff
@@ -133,6 +135,7 @@ jobs:
             --app-version "${{ needs.metadata.outputs.versionNum }}"
             --java-options "-Dfile.encoding=\"utf-8\""
             --java-options "-Dcryptomator.logDir=\"~/Library/Logs/Cryptomator\""
+            --java-options "-Dcryptomator.pluginDir=\"~/Library/Application Support/Cryptomator/Cryptomator/Plugins\""
```

Comment:
`--java-options "-Dcryptomator.pluginDir=\"~/Library/Application Support/Cryptomator/Plugins\""`

There is a typo: one _Cryptomator_ should be enough.

Reply:
Oh right, did see this comment a little late, but will fix it in a follow up commit!

Reply:
No problem. 🙂

Reply:
fixed in aef1bf8. :)

next time, feel free to add those remarks as a reviews instead of of standalone comments, so they draw more attention